### PR TITLE
Force endpoint

### PR DESCRIPTION
**Pull request by peterschubert**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -57,15 +57,27 @@ Please check --help for more details.
...
+`/{content-type}/{uuid}`
+###POST
+Creates and forwards a CombinedPostPublicationEvent to the queue for the provided UUID.
+
+Supported content types: `article`, `blogPost`, `video`
+Request body should be empty.
+
+Returns 200 if the message was published successfully
+
+Returns 400 for unrecognized content-type or and invalid uuid
```
Comment (peterschubert) [inline Readme.md]: done

File: main.go [hunk 2]
```diff
@@ -215,9 +227,72 @@ func routeRequests(port *string, healthService *HealthcheckHandler) {
...
 	if err := http.ListenAndServe(":"+*port, r); err != nil {
 		logrus.Fatalf("Unable to start: %v", err)
 	}
 }
+
+type requestHandler struct {
+	processor processor.Processor
+}
+
+func (handler *requestHandler) postMessage(writer http.ResponseWriter, request *http.Request) {
```
Comment (peterschubert) [inline main.go]: done

File: main.go [hunk 3]
```diff
@@ -215,9 +227,72 @@ func routeRequests(port *string, healthService *HealthcheckHandler) {
...
+	}
+
+	platform := platformV1
+
+	if contentType == contentTypeVideo {
+		platform = platformVideo
+	}
+
+	err := handler.processor.ForceMessagePublish(uuid, platform)
+	if err != nil {
```
Comment (peterschubert) [inline main.go]: done

File: main.go [hunk 4]
```diff
@@ -215,9 +227,72 @@ func routeRequests(port *string, healthService *HealthcheckHandler) {
...
+		writer.WriteHeader(http.StatusInternalServerError)
+	} else {
+		writer.WriteHeader(http.StatusOK)
+	}
+}
+
+func isValidContentType(contentType string) bool {
```
Comment (peterschubert) [inline main.go]: moved with the handler

File: processor/msg_processor.go [hunk 5]
```diff
@@ -221,8 +221,8 @@ func includes(array []string, element string) bool {
 }
 
 func getPlatformVersion(str string) string {
-	if strings.Contains(str, "brightcove") {
-		return "brightcove"
+	if strings.Contains(str, "video") {
+		return "next-video"
```
Comment (tosan88) [inline processor/msg_processor.go]: Someone said "video"? :)

Comment (peterschubert) [inline processor/msg_processor.go]: done

**Pull request merged**